src/TestchainClient.js

```diff
@@ -0,0 +1,72 @@
+import TestchainService from './TestchainService';
+import SnapshotService from './SnapshotService';
+import EventHandlerService from './EventHandlerService';
+
+class TestchainClient {
+  constructor() {}
+}
+
+// Add additional services to the superCtors array
+const client = inheritsMultiple(TestchainClient, [
```

Comment:
Best implementation I found to create a class that inherits from multiple classes. Allows consumers to use `service.myServiceMethod()` e.g. the TC Dashboard. It's a little strange, what do you guys think?

Reply:
I think if we really want to mix everything into a single class then we might as well just write a single class, and if we want to break it into smaller files we can use object literals and merge them. But that comes with the downsides of increased coupling. 

If all we’re trying to accomplish is having a cleaner API that doesn’t require callers to know about individual services, then something like `delegateToServices` from Maker.js (perhaps more automated) would suffice.

Reply:
Actually I did have an implementation working that I borrowed from `delegateToServices` before I got sucked into the rabbit hole of multiple class inheritance. Maybe that's the better option, I'll spin it up tomorrow and see what people think.